python/run-tests.py

```diff
@@ -267,8 +267,8 @@ def main():
         python_implementation = subprocess_check_output(
             [python_exec, "-c", "import platform; print(platform.python_implementation())"],
             universal_newlines=True).strip()
-        LOGGER.debug("%s python_implementation is %s", python_exec, python_implementation)
-        LOGGER.debug("%s version is: %s", python_exec, subprocess_check_output(
+        LOGGER.info("%s python_implementation is %s", python_exec, python_implementation)
+        LOGGER.info("%s version is: %s", python_exec, subprocess_check_output(
```

Comment:
I'm gonna check the output and piggyback this as well if it doesn't look so noisy.

Reply:
Looks not so noisy. 6 extra lines.

```
========================================================================
Running PySpark tests
========================================================================
Running PySpark tests. Output is in /home/jenkins/workspace/SparkPullRequestBuilder/python/unit-tests.log
Will test against the following Python executables: ['python3.6', 'python2.7', 'pypy']
Will test the following Python modules: ['pyspark-core', 'pyspark-sql', 'pyspark-streaming', 'pyspark-mllib', 'pyspark-ml']
python3.6 python_implementation is CPython
python3.6 version is: Python 3.6.8 :: Anaconda, Inc.
python2.7 python_implementation is CPython
python2.7 version is: Python 2.7.13 :: Anaconda 2.1.0 (64-bit)
pypy python_implementation is PyPy
pypy version is: Python 2.7.9 (9c4588d731b7fe0b08669bd732c2b676cb0a8233, Apr 09 2015, 02:17:39)
[PyPy 2.5.1 with GCC 4.4.7 20120313 (Red Hat 4.4.7-11)]
Starting test(pypy): pyspark.sql.tests.test_arrow
...
```